OpenRA.Game/Network/OrderManager.cs

```diff
@@ -83,6 +80,14 @@ public void StartGame()
 			if (GameStarted)
 				return;
 
+			// Special case for shellmap
+			if (LobbyInfo.Clients.Count == 0)
```

Comment:
What if we added somethign like this to `Game.StartGame` instead?
```csharp
if (type == WorldType.Shellmap)
{
	// Add a local spectator client to represent the player,
    // who is controlling the map via scripted orders
	OrderManager.LobbyInfo.Clients.Add(new Session.Client
	{
		Index = OrderManager.Connection.LocalClientId,
		Name = Settings.Player.Name,
		PreferredColor = Settings.Player.Color,
		Color = Settings.Player.Color,
		Faction = "Random",
		SpawnPoint = 0,
		Team = 0,
		State = Session.ClientState.Ready
	});
}
```

Reply:
Looked like we could just add it no LoadShellMap().